source-academy/frontend
# Integrate Java type checker and compiler into frontend

### Description



### Type of change


### How to test


Input a simple Java program (such as the following) into the code editor and click on the `Run` button:
```
public class Main {
    public static void main(String[] args) {
        System.out.println("Hello world");
    }
}
```
Make sure to choose `Java` as the language option.

### Checklist
